packages/codemods/package.json

```diff
@@ -25,7 +25,8 @@
   "devDependencies": {
     "@fluentui/eslint-plugin": "*",
     "@fluentui/scripts-jest": "*",
-    "@fluentui/scripts-tasks": "*"
+    "@fluentui/scripts-tasks": "*",
+    "typescript": "5.3.3"
```

Comment:
we don't wanna maintain various TS versions in monorepo. for short term it's ok 
-> please create issue to bump ts-morph in this package to get rid of this TS dep
-> if the upgrade of ts-morph is easy enough we can do it as part of this PR

_sidenote:_ this package should be probably removed completely as it only use was for v8 AFAIK and has not been updated for quite some time , so if the efforts of bumping ts-morph are huge better investment is to remove this package completely

Reply:
I checked changelog, most of the breaking changes coming from typescript updates, other relate to API that seem to be not used inside our package. Updated to v23, all tests are passing, so I guess it's ok 🤔